package.json

```diff
@@ -17,7 +17,7 @@
 		"deploy": "yarn run lint && yarn test && node tools/deploy.mjs",
 		"deploy-all": "yarn run lint && yarn test && node tools/deploy-all.mjs",
 		"start-watch": "concurrently --kill-others \"yarn run start\" \"yarn run watch\"",
-		"watch": "yarn run ajv-hack && node --openssl-legacy-provider tools/watch.js",
+		"watch": "yarn run ajv-hack && node tools/watch.js",
```

Comment:
Don't change this, upgrade to Node 17 instead.

Reply:
didnt mean to commit that oops